express-notes-api/index.js

```diff
@@ -0,0 +1,99 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+const data = require('./data');
+const notes = data.notes;
+
+app.use(express.json());
+
+app.get('/api/notes', (req, res) => {
+  const arr = [];
+  for (const property in notes) {
+    arr.push(notes[property]);
+  }
+  res.json(arr);
+  res.sendStatus(200);
```

Comment:
This is technically sending 2 responses, surprised it doesn't cause an error. Also when using a status `200` you don't need to set it, `200` is the default status code. So here you just need:

```JAVASCRIPT
res.json(arra);
```